package.json

```diff
@@ -1,6 +1,7 @@
 {
-    "name": "@microsoft.azure/autorest.typescript",
-    "version": "6.0.0-preview.1",
+    "name": "@autorest/typescript",
+    "version": "6.0.0",
+    "private": true,

```

Comment:
Setting the package to `private` until we're ready to publish on npm.  Happy to remove it if folks feel it is unnecessary.